web/yo/app/scripts/controllers/gene.js

```diff
@@ -354,13 +375,13 @@ angular.module('oncokbApp')
             $scope.displayCheck = function(uuid, reviewObj) {
                 // regular mode check
                 if (!$rootScope.reviewMode) {
-                    if (reviewObj && reviewObj.get('removed')) {
+                    if (reviewObj && reviewObj.removed === true) {
```

Comment:
Could just be `if (reviewObj && reviewObj.removed)`?

Reply:
This will be taken care of in the Review related refactor.